.github/workflows/deploy.nightly.devnet.yml

```diff
@@ -0,0 +1,193 @@
+          TF_VAR_aws_profile: ${{ secrets.TF_VAR_AWS_PROFILE }}
+          TF_VAR_create_ssh_key: ${{ secrets.TF_VAR_CREATE_SSH_KEY }}
+          TF_VAR_devnet_key_value: ${{ secrets.TF_VAR_DEVNET_KEY_VALUE }}
+          TF_VAR_jumpbox_ssh_access: ${{ secrets.TF_VAR_JUMPBOX_SSH_ACCESS }}
+      - name: Terraform Apply
+        id: apply
+        run: terraform apply -auto-approve tfplan
+        env:
+          TF_VAR_explorer_rds_master_password: ${{ secrets.TF_VAR_EXPLORER_RDS_MASTER_PASSWORD }}
+          TF_VAR_deployment_name: ${{ secrets.TF_VAR_DEPLOYMENT_NAME }}
+          TF_VAR_company_name: ${{ secrets.TF_VAR_COMPANY_NAME }}
+          TF_VAR_owner: ${{ secrets.TF_VAR_OWNER }}
+          TF_VAR_aws_profile: ${{ secrets.TF_VAR_AWS_PROFILE }}
+          TF_VAR_create_ssh_key: ${{ secrets.TF_VAR_CREATE_SSH_KEY }}
+          TF_VAR_devnet_key_value: ${{ secrets.TF_VAR_DEVNET_KEY_VALUE }}
+          TF_VAR_jumpbox_ssh_access: ${{ secrets.TF_VAR_JUMPBOX_SSH_ACCESS }}
+      - name: Install ansible / botocore / boto3
+        run: |
+          python3 -m pip install --user ansible
+          python3 -m pip install boto3 botocore
+      - name: Run Ansible
+        working-directory: ansible
+        run: |
+          touch password.txt
+          echo "${{ secrets.VAULT_PASSWORD_FILE }}" > password.txt
+          mkdir ~/cert
+          touch ~/cert/2022-07-05-devnets-generic.key
+          echo "${{ secrets.ANSIBLE_PRIVATE_SSH }}" > ~/cert/2022-07-05-devnets-generic.key
+          chmod 600 ~/cert/2022-07-05-devnets-generic.key
+          ansible-inventory --graph
+          ansible-galaxy install -r requirements.yml
+          ansible-playbook --inventory inventory/aws_ec2.yml --vault-password-file=password.txt --extra-vars "@local-extra-vars.yml" site.yml
+      - name: Set rpc url value
+        id: url
+        run: |
+          touch rpc.txt
+          terraform output -raw aws_lb_ext_validator_domain | grep -o -E '^ext[^:]*' > rpc.txt 
+      - uses: actions/upload-artifact@v3
+        with:
+          name: rpc-url
+          path: rpc.txt
+
+  loadtest1:
+    needs: build
+    uses: ./.github/workflows/loadtest.yml
+    name: Load Test Nightly Build - multiple_EOA
+    secrets:
+      SLACK_PERFORMANCE_WEBHOOK_URL: ${{ secrets.SLACK_PERFORMANCE_WEBHOOK_URL }}
+      DD_API_KEY: ${{ secrets.DD_API_KEY }}
+      LOADTEST_MNEMONIC: ${{ secrets.LOADTEST_MNEMONIC }}
+    with:
+      environment: ${{ inputs.environment }}
+      scenario: multiple_EOA
+
+  loadtest2:
+    needs: loadtest1
+    uses: ./.github/workflows/loadtest.yml
+    name: Load Test Nightly Build - multiple_ERC20
+    secrets:
+      SLACK_PERFORMANCE_WEBHOOK_URL: ${{ secrets.SLACK_PERFORMANCE_WEBHOOK_URL }}
+      DD_API_KEY: ${{ secrets.DD_API_KEY }}
+      LOADTEST_MNEMONIC: ${{ secrets.LOADTEST_MNEMONIC }}
+    with:
+      environment: ${{ inputs.environment }}
+      scenario: multiple_ERC20
+
+  destroy_devnet:
+    needs: [loadtest1, loadtest2]
```

Comment:
Sorry for my ignorance, but hope that `destroy_devnet` is going to be triggered even if `loadtest1` and/or `loadtest2` fail?